Tests/AlicerceTests/Extensions/Foundation/OptionalTests.swift

```diff
@@ -7,7 +7,7 @@ final class OptionalTests: XCTestCase {
     func testThen_UsingNonOptionalValue_ShouldUnwrapTheValue() {
```

Comment:
This whole test no longer makes sense without `then`

Reply:
You're right 🙈